### PR TITLE
feature/mx-1686 wrap up blueant

### PR DESCRIPTION
**Pull request by erichesse**
# PR Context
regex: https://regex101.com/r/EVN4ad/1
-wrap up blueant
-based on mex-assets branch `feature/mx-1686-wrap-up-blueant`

### CODE REVIEW

File: mex/extractors/blueant/transform.py [hunk 1]
```diff
@@ -68,29 +69,33 @@ def transform_blueant_sources_to_extracted_activities(
...
         ]
-        if not contact and responsible_unit:
-            contact.append(responsible_unit)
+        if not contact and department_id:
+            contact.append(department_id)
+        source_name = re.sub(
+            r"[\d*_]+|[FG\d* ]+[- ]+", "", source.name
+        )  # strip according to mapping
+        if source_name not in activity.title[0].mappingRules[1].forValues:
+            title = source_name
```
Comment (cutoffthetop) [inline mex/extractors/blueant/transform.py:87]:
```suggestion
            title = source_name
        else:
            title = []
```

File: mex/extractors/blueant/transform.py [hunk 2]
```diff
@@ -68,29 +69,33 @@ def transform_blueant_sources_to_extracted_activities(
...
         # get contact employee or fallback to unit
         contact = person_stable_target_ids_by_employee_id[
             source.projectLeaderEmployeeId
         ]
-        if not contact and responsible_unit:
-            contact.append(responsible_unit)
+        if not contact and department_id:
+            contact.append(department_id)
+        source_name = re.sub(
+            r"[\d*_]+|[FG\d* ]+[- ]+", "", source.name
```
Comment (cutoffthetop) [inline mex/extractors/blueant/transform.py:84]: ZV prefix is ok though? or will it be skipped anyway

**Review by cutoffthetop — CHANGES_REQUESTED** [review]
looks good, just some questions on the title

File: mex/extractors/blueant/transform.py (hunk 2 above)
Comment (erichesse) [reply mex/extractors/blueant/transform.py:84]: the rule is `Delete underscore (_) and number_ if used as prefix in a label`, so I think we can keep `ZV` here. Plus I got the check mark in the ticket table for the title field. So I think I'm covered here.

**Comment by erichesse** [comment]
your suggestion I signed of in transform.py:89 is causing a mypy error @cutoffthetop 
You sure, this is needed here? The None case is covered below anyway

**Review by cutoffthetop — APPROVED** [review]

**Pull request merged**